test/suites/Tests-7-ECMA.js

```diff
@@ -0,0 +1,348 @@
+     });
+     
+     test("7.2 White space- UTF spaces aceptation 18", function(){
+         var result = puma.evalPuma("var a= \"a\u202Fb\"");
+         result.makeValue();
+         ok(result.success && "a\u202Fb" === result.value, "Passed!");
+     });
+     
+     test("7.2 White space- UTF spaces aceptation 19", function(){
+         var result = puma.evalPuma("var a= \"a\u205Fb\"");
+         result.makeValue();
+         ok(result.success && "a\u205Fb" === result.value, "Passed!");
+     });
+     
+     test("7.2 White space- UTF spaces aceptation 20", function(){
+         var result = puma.evalPuma("var a= \"a\u3000b\"");
+         result.makeValue();
+         ok(result.success && "a\u3000b" === result.value, "Passed!");
+     });
+     
+     test("7.6 Identifier names and identifiers 1", function(){
+         var result = puma.evalPuma("var $a$b$ = 1;");
+         result.makeValue();
+         ok(result.success && 1 === result.value, "Passed!");
+     });
+     
+     test("7.6 Identifier names and identifiers 2", function(){
+         var result = puma.evalPuma("var _a_b_ = 1;");
+         result.makeValue();
+         ok(result.success && 1 === result.value, "Passed!");
+     });
+     
+     //Future reserved words works unless used with "use strict"
+     test("7.6.1 Future reserved words 1", function(){
+         var result = puma.evalPuma("var implements = 1;");
+         result.makeValue();
+         ok(result.success && 1 === result.value, "Passed!");
+     });
+     
+     test("7.6.1 Future reserved words 1", function(){
+         var result = puma.evalPuma("var private = 1;");
+         result.makeValue();
+         ok(result.success && 1 === result.value, "Passed!");
+     });
+     
+     test("7.6.1 Future reserved words 1", function(){
+         var result = puma.evalPuma("var public = 1;");
+         result.makeValue();
+         ok(result.success && 1 === result.value, "Passed!");
+     });
+     
+     test("7.6.1 Future reserved words 1", function(){
+         var result = puma.evalPuma("var interface = 1;");
+         result.makeValue();
+         ok(result.success && 1 === result.value, "Passed!");
+     });
+     
+     test("7.6.1 Future reserved words 1", function(){
+         var result = puma.evalPuma("var package = 1;");
+         result.makeValue();
+         ok(result.success && 1 === result.value, "Passed!");
+     });
+     
+     test("7.6.1 Future reserved words 1", function(){
+         var result = puma.evalPuma("var protected = 1;");
+         result.makeValue();
+         ok(result.success && 1 === result.value, "Passed!");
+     });
+     
+     test("7.6.1 Future reserved words 1", function(){
+         var result = puma.evalPuma("var static = 1;");
+         result.makeValue();
+         ok(result.success && 1 === result.value, "Passed!");
+     });
+     
+     /*
+     //This tests fail. Uncaught Error: Unexpected token
+     test("7.6.1 Future reserved words 1", function(){
```

Comment:
I think you can use "skip" to avoid having to comment the code for a test that is not passing